ansible/roles/knative/defaults/main.yml

```diff
@@ -12,7 +12,8 @@ serverless_operator_subscription_name: serverless-operator
 serverless_operator_subscription_channel: 'stable'
 serverless_operator_csv_prefix: serverless-operator
 serverless_operator_deployment_name: knative-operator
-namespace_serverless_operator: openshift-serverless
```

Comment:
changed it back to openshift-operators since for some reason when the ansible script attempts to create the openshift-serverless namespace, an error is received saying that projects starting with "openshift-" cannot be created